server/lib/nl/nl_data_spec_next.py

```diff
@@ -638,25 +642,12 @@ def _add_chart_to_utterance(chart_type: ChartType, state: PopulateState,
       "include_percapita": chart_vars.include_percapita,
       "title": chart_vars.title,
   }
-  if len(chart_vars.svs) < 2 or chart_type == ChartType.SCATTER_CHART:
-    ch = ChartSpec(chart_type=chart_type,
-                   svs=chart_vars.svs,
-                   places=places,
-                   utterance=state.uttr,
-                   attr=attr)
-    state.uttr.chartCandidates.append(ch)
-    return True
-
-  start_index = 0
-  while start_index < len(chart_vars.svs):
-    l = min(len(chart_vars.svs) - start_index, _MAX_VARS_PER_CHART)
-    ch = ChartSpec(chart_type=chart_type,
-                   svs=chart_vars.svs[start_index:start_index + l],
-                   places=places,
-                   utterance=state.uttr,
-                   attr=attr)
-    start_index += l
-    state.uttr.chartCandidates.append(ch)
```

Comment:
To confirm, before there could be several charts, now they are all merged into one?

Reply:
Correct.  Yes, before we breakup SVs across multiple time-line charts. Guha wanted them all in one long Bar chart.